src/components/TextArea/TextArea.tsx

```diff
@@ -28,40 +34,70 @@ export type TextAreaProps = {
   onKeyDown?: React.KeyboardEventHandler<HTMLTextAreaElement | HTMLInputElement>;
   /** Callback fired when the `input` value typed is changed */
   onInput?: React.EventHandler<any>;
-} & Pick<TextInputBaseProps, 'status'> &
+} & Pick<HTMLTextAreaElement, 'cols' | 'rows'> &
+  Pick<TextInputBaseProps, 'status' | 'label'> &
   TestProps;
 
 const TextArea = React.forwardRef<HTMLTextAreaElement, TextAreaProps>((props, ref) => {
   const {
     id = undefined,
-    placeholder = '',
+    label,
+    placeholder,
     isRequired = false,
+    isReadOnly,
     isDisabled,
+    status,
     isResizeEnabled = true,
```

Comment:
is this the default behavior? are we sure?

Reply:
after discussion with the Design Team, the implied that this is the default expected behavior.